crates/sui-indexer/src/models/objects.rs

```diff
@@ -65,6 +89,16 @@ impl Object {
     ) -> Self {
         let (owner_type, owner_address, initial_shared_version) =
             owner_to_owner_info(&o.owner.expect("Expect the owner type to be non-empty"));
+
+        let bcs = match o.bcs.clone().expect("Expect BCS data to be non-empty") {
+            SuiRawData::MoveObject(o) => vec![NamedBcsBytes(OBJECT.to_string(), o.bcs_bytes)],
```

Comment:
chatted offline, eventually we can split `package` data out of this table, then the bcs can just be one `bytea`.